pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+// are on one line! For multiple line or blocks that you want to ignore use ---.
+// Any context after a --- is ignored.
+//
+// Those methods can be generated by using hack/update-generated-swagger-docs.sh
+
+// AUTO-GENERATED FUNCTIONS START HERE
+func (APIVersion) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "An APIVersion represents a single concrete version of an object model.",
+		"name":     "Name of this version (e.g. 'v1').",
+		"apiGroup": "The API group to add this object into. Default 'experimental'.",
+	}
+}
+
+func (AWSElasticBlockStoreVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "AWSElasticBlockStoreVolumeSource represents a Persistent Disk resource in AWS.\n\nAn Amazon Elastic Block Store (EBS) must already be created, formatted, and reside in the same AWS zone as the kubelet before it can be mounted. Note: Amazon EBS volumes can be mounted to only one instance at a time.",
+		"volumeID":  "Unique ID of the persistent disk resource in AWS (Amazon EBS volume). More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+		"fsType":    "Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+		"partition": "The partition in the volume that you want to mount. If omitted, the default is to mount by volume name. Examples: For volume /dev/sda1, you specify the partition as \"1\". Similarly, the volume partition for /dev/sda is \"0\" (or you can leave the property empty).",
+		"readOnly":  "Specify \"true\" to force and set the ReadOnly property in VolumeMounts to \"true\". If omitted, the default is \"false\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+	}
+}
+
+func (Binding) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "Binding ties one object to another. For example, a pod is bound to a node by a scheduler.",
+		"metadata": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"target":   "The target object that you want to bind to the standard object.",
+	}
+}
+
+func (Capabilities) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "Adds and removes POSIX capabilities from running containers.",
+		"add":  "Added capabilities",
+		"drop": "Removed capabilities",
+	}
+}
+
+func (ComponentCondition) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":        "ComponentCondition contains information for the condition of a component.",
+		"type":    "Type of component condition, currently only Healthy.",
+		"status":  "Current status of this component condition, one of True, False, Unknown.",
```

Comment:
Revision:
"Status of the condition for a component. Valid values:\n\nFor "/Healthy"/: "/True"/, "/False"/, or "/Unknown"/